lib/packer.ts

```diff
@@ -47,7 +47,7 @@ function hasAnyStore(record: FileRecord) {
 interface PackerOptions {
   records: FileRecords;
   entrypoint: string;
-  bytecode: string;
+  bytecode: boolean;
```

Comment:
Why this change?

Reply:
It was a left-over of when I was trying to understand the code to get this PR to work.